packages/orchestration/src/chain-info.js

```diff
@@ -65,6 +65,35 @@ const knownChains = /** @satisfies {Record<string, ChainInfo>} */ (
 
 /** @typedef {typeof knownChains} KnownChains */
 
+/**
+ * Utility to reverse connection info perspective.
+ *
+ * @param {IBCConnectionInfo} connInfo
+ * @returns {IBCConnectionInfo}
+ */
+const reverseConnInfo = connInfo => {
```

Comment:
Do we also need something similar for `ChainHub.getConnectionInfo`? It seems we'll currently get the same response if we did do `getConnectionInfo(id1, id2)` versus `getConnectionInfo(id2, id1)`, which may be unexpected to the caller cc @turadg @dckc

Reply:
hm. good point

Reply:
@dckc see https://github.com/Agoric/agoric-sdk/pull/9718